portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -45,8 +51,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
+    // Set nickname to "Anonymous" if none specified
+    if (nickname == null || nickname.isEmpty()) {
+      nickname = "Anonymous";
+    }
+
     // Create new entity for the comment and store in Datastore
     Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("email", email);
+    commentEntity.setProperty("nickname", nickname);
     commentEntity.setProperty("text", text);
     commentEntity.setProperty("timestamp", timestamp);
```

Comment:
If these strings are used in multiple locations, it may make more sense to create a common constants file that all instances refer to rather than having multiple instances of the same string throughout several files. That would make refactoring/changing the value of the string much easier, as only 1 location would need to be updated: the constant location.

Reply:
Oh that's a good idea. I'll make a Constants file in a separate pull request.